tests/test_lodcm.py

```diff
@@ -331,77 +318,61 @@ def test_get_energy_si(fake_energy_si, monkeypatch):
                return_value=(1, 1, 1)):
         # offset of 23, motor position 0
         # current th1.wm() should be 23
+        assert energy.th1.wm() == 23
+        res = energy.get_energy()
+        assert res == 5.059840436879476
+
+
+def test_forward_si(fake_energy_si, monkeypatch):
+    energy = fake_energy_si
+
+    energy.th1.set_current_position(-23)
+    with patch("pcdsdevices.lodcm.LODCMEnergySi.get_reflection",
+               return_value=(1, 1, 1)):
         res = energy.get_energy()
         assert res == 5.059840436879476
+        res = energy.forward(5.059840436879476)
```

Comment:
The forward function is giving back what the 5 motor positions should be when the pseudo_pos is 5.0598....
It seems like it gives me back the correct stuff, however when I try to `move` the energy - it does not change the motor positions - this could be because of me not setting up correctly the `fake_lodcm` or the `fake_energy` objects.... 

Is there another easier way to check if the `move` is actually working somehow?

Reply:
The motor's you're using in the test aren't actually motors, they are just collections of PVs that get their `user_setupoint` set when you try to move them and do no other changes. So as long their setpoints are changing I think it works.

If you want to see them "working" we need to actually sub in a simulated motor object instead of just replacing all the PVs with fake PVs like is done currently.

Reply:
Got it, I'd have to use the good old `FastMotor`  for every IMS I have in my case and maybe others.... ok, thank you